notify-send.sh

```diff
@@ -19,6 +19,19 @@
 # Desktop Notifications Specification
 # https://developer.gnome.org/notification-spec/
 
+DIR=`dirname "$0"`
+
+# set -x
+
+run_action_handler () {
+  action=`ps axu | grep "notify-action.sh" | grep -v "grep"`
+  if [[ -z "$action" ]] ; then
+    echo "run_action_handler"
+    ($DIR/notify-action.sh 2> /dev/null &)
+  fi
+}
+run_action_handler
```

Comment:
Can we try to run action daemon only if `--action` is specified?

Reply:
Done